app/cdap/components/ConnectionList/Components/TabLabelCanBrowse/styles.ts

```diff
@@ -31,13 +31,17 @@ export const useStyles = makeStyles({
     height: '24px',
   },
   labelStyles: {
-    maxWidth: '192px',
+    maxWidth: '153px',
     whiteSpace: 'nowrap',
     fontSize: '16px',
```

Comment:
use fontSize: 16 instead of fontSize: '16px',

Reply:
Done.